### DISCUSSION TITLE
Check-In auch via Zugnummer

### DISCUSSION THREAD

**Discussion by PatrikRic** [Traewelling/traewelling#2438] [Ideas / Feature Requests]
Hey, ich bin Zub und würde es viel leichter finden kurz die Zugnummer einzugeben und dann den Start und Endbahnhof auszuwählen. Gerade bei großen Bahnhöfen wie München Hbf ist das ein gefriemel bis man den richtigen Zug hat

**Comment by MrKrisKrisu** [comment]
Hey! Das kann ich verstehen, jedoch haben wir mit den Schnittstellen, die wir aktuell nutzen leider keine Möglichkeit einen Zug nach Zugnummer zu suchen. 

(duplicate of #2014)

**Reply by Eperoais** [reply]
Die API Sollte dies ermöglichen:
https://v6.db.transport.rest/trips/?query=Linien/Zug/Fahrtnummer zeigt die verbindungen mit der Linien/Zug/Fahrtnummer an (In einem gewissen Zeitrahmen)
Ausser Träwelling benutzt noch die veraltete version 5 die man nicht mehr benutzen sollte.

**Reply by MrKrisKrisu** [reply]
Bitte weitere Interaktion nach #2014 verschieben, da dies ein Duplikat ist.

(Aber dazu noch: Träwelling nutzt aus Gründen (siehe #2032) noch die v5.)
